src/main/java/org/sagebionetworks/bridge/cache/CacheKey.java

```diff
@@ -29,7 +29,10 @@ public final static boolean isPublic(String key) {
         }
         return false;
     }
-
+    
+    public static final CacheKey metricsKey(String requestId) {
+        return new CacheKey(requestId, "Metrics"); 
+    }
```

Comment:
Does it make sense to use Redis for this? A request is handled by at most a single host (and oftentimes a single thread). Would it make more sense to store metrics in the RequestContext?

Reply:
It definitely makes sense. Changing so metrics is held by RequestContext. I also renamed RequestContext.getId() to getRequestId() so it matches usage elsewhere and it's clear that these two values are the same in a request.